tools/kyma-installer/kyma.Dockerfile

```diff
@@ -1,7 +1,4 @@
-ARG INSTALLER_VERSION="c74d3125"
-ARG INSTALLER_DIR=eu.gcr.io/kyma-project
-FROM $INSTALLER_DIR/kyma-operator:$INSTALLER_VERSION
-
+FROM eu.gcr.io/kyma-project/kyma-operator:master-c74d3125
```

Comment:
shouldn't it be 1.11.0-rc1 as well?

Reply:
This line is modified according to the documentation.
There is no ```eu.gcr.io/kyma-project/kyma-operator:1.11.0-rc1``` in the registry.